common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -366,18 +316,15 @@
       .control {
         @extend %video-control;
 
-        .icon-fallback-img {
-
-          .icon {
-            // if the icon font doesn't render, we need to provide dimensions for the svg's
-            width: 1em;
-            height: 1em;
+        .icon {
```

Comment:
If there is no width, the alignment is slightly off by 2 pixels, as tt will fallback to FA font style definition which is 14px. You can see the difference in the browser console. Your call.